packages/server/lib/browsers/electron.ts

```diff
@@ -465,6 +465,18 @@ export = {
     throw new Error('Attempting to connect to existing browser for Cypress in Cypress which is not yet implemented for electron')
   },
 
+  validateLaunchOptions (launchOptions: typeof utils.defaultLaunchOptions) {
+    const options: string[] = []
+
+    if (Object.keys(launchOptions.env).length > 0) options.push('env')
+
+    if (launchOptions.args.length > 0) options.push('args')
+
+    if (options.length > 0) {
+      errors.warning('BROWSER_UNSUPPORTED_LAUNCH_OPTION', 'electron', options)
```

Comment:
@ryanjduffy sorry last comment here before approval... how can I actually pass something to this? I want to verify this error actually shows up, but I couldn't figure out how.

Reply:
Something like this should do it!

```
      on("before:browser:launch", (browser, launchOptions) => {
        launchOptions.env = {
          DEBUG: "whatever",
        };

        return launchOptions;
      });
```

<img width="551" alt="image" src="https://user-images.githubusercontent.com/788456/191327270-ee9dc6bb-d634-410f-b140-da7a65ae9c3d.png">